src/lib/dateFormatter.ts

```diff
@@ -31,7 +31,7 @@ export const dateFormatter = (dateString: string) => {
 
 	return `${date.getDate()} ${
 		monthConverter[date.getMonth()]
-	} ${date.getFullYear()} ${date.getHours()}:${
+	} ${date.getFullYear()} ${date.getHours()}.${
 		(date.getMinutes() < 10 ? '0' : '') + date.getMinutes()
```

Comment:
`String(date.getMinutes()).padLeft(2, "0")` ?

Reply:
That's really nice! I'll add it